website/project/model.py

```diff
@@ -248,13 +288,16 @@ def create(cls, auth, **kwargs):
         else:
             comment.root_target = comment.target
 
-        if isinstance(comment.root_target.referent, Node):
-            comment.page = Comment.OVERVIEW
-        elif isinstance(comment.root_target.referent, StoredFileNode):
-            log_dict['file'] = {'name': comment.root_target.referent.name, 'url': comment.get_comment_page_url()}
-            comment.page = Comment.FILES
-        else:
+        page = getattr(comment.root_target.referent, 'root_target_page', None)
+        if not page:
             raise ValueError('Invalid root target.')
+        comment.page = page
+
+        if comment.page == Comment.FILES:
```

Comment:
Ah, I see that the same params are added for all types of comment-related logs (create, edit, etc.). Therefore, better to use a more generic name for the hook, .e.g `get_extra_log_params(self, comment)`.